webui/react/src/pages/WorkspaceDetails/WorkspaceMembers.tsx

```diff
@@ -41,14 +41,14 @@ interface Props {
 interface GroupOrMemberActionDropdownProps {
   fetchMembers: () => void;
   name: string;
-  roleId: number;
+  roleIds: [number];
```

Comment:
change this as array cuz [this line](https://github.com/determined-ai/determined/pull/5640/files#diff-a9a21494cf20b899ce8dc5a2baa98d4b53f28afe1d3ea1ab63d30895b43b14b3L62) is recalculated every render, and the prop doesnt have to be just `number`